rebar.config.script

```diff
@@ -1,7 +1,7 @@
 IsRebar3 = case application:get_key(rebar, vsn) of
                {ok, VSN} ->
                    [VSN1 | _] = string:tokens(VSN, "-"),
-                   [Maj, Min, Patch] = string:tokens(VSN1, "."),
+                   [Maj, Min, Patch | _] = string:tokens(VSN1, "."),
```

Comment:
Also `[VSN1 | _] = string:tokens(VSN, "-")` on the line above is not needed if matching on the front of the list.